packages/sanes-browser-extension/src/routes/wallet/components/SidePanel/PanelDrawer/components/Settings.tsx

```diff
@@ -0,0 +1,66 @@
+import { Block, Image, List, ListItem, ListItemText, makeStyles } from "medulas-react-components";
+import React from "react";
+
+import { Views } from "..";
+import greenRightChevron from "../../../../assets/chevronRightGreen.svg";
+
+const useStyles = makeStyles({
+  listRoot: {
+    border: "none",
+  },
+  listButton: {
+    borderBottom: "1px solid #e0e0e0",
+  },
+});
+
+interface Props {
+  updateCurrentView: (newView: Views) => void;
+}
+
+const Settings = ({ updateCurrentView }: Props): JSX.Element => {
+  const classes = useStyles();
+  const listClasses = {
+    root: classes.listRoot,
+  };
+  const listItemClasses = {
+    button: classes.listButton,
+  };
+
+  const goToNetworks = (): void => updateCurrentView(Views.Networks);
+  const goToAbout = (): void => updateCurrentView(Views.About);
+  const goToRecoveryWords = (): void => updateCurrentView(Views.RecoveryWords);
+  const goToDeleteWallet = (): void => updateCurrentView(Views.DeleteWallet);
+
+  return (
+    <Block width="100%">
+      <List component="nav" classes={listClasses}>
+        <ListItem button key={Views.Networks} onClick={goToNetworks} classes={listItemClasses}>
```

Comment:
Can you move `ListItems` to separate component? Because there is Block wrapper inside and it would be great not to copy it between items.